lisa/util/subclasses.py

```diff
@@ -70,7 +70,9 @@ def load_typed_runbook(self, raw_runbook: Any) -> T_BASECLASS:
         sub_type = self.get(type_name)
         if sub_type is None:
             raise LisaException(
-                f"cannot find subclass '{type_name}' of {self._base_type.__name__}"
+                f"cannot find subclass '{type_name}' of {self._base_type.__name__}. "
+                f"Supported types include: {list(self.keys())}. "
```

Comment:
I added this just in case the problem is just a spelling mistake in the runbook file.

Reply:
It's great. How about below more?
1. As the three places are the same, create a method called `def _get_sub_type(self, type_name:str) -> T_BASECLASS`. So, the logic can be shared.
2. Let the message be more actionable. For example, "Check the spell of subtype name, and make sure it's added in mixin_modules.py or customized mixin files."